tachyon/zk/lookup/BUILD.bazel

```diff
@@ -17,6 +17,11 @@ tachyon_cc_library(
     deps = ["//tachyon/base/json"],
 )
 
+tachyon_cc_library(
+    name = "opening_point_set",
+    hdrs = ["opening_point_set.h"],
+)
+
```

Comment:
I don't think this opening point set is used by that many lookup implementations! Why don't you make `LogUpProver` depend on `//tachyon/zk/lokoup/halo2:opening_point_set`?